In a Debian installation, not all directories in @INC need exist (the
site directories for example are created on demand).

Suggested by Joey Hess <joeyh@debian.org>.

diff -ur perl-5.6.1.orig/utils/perldoc.PL perl-5.6.1/utils/perldoc.PL
--- perl-5.6.1.orig/utils/perldoc.PL	Fri Apr  6 14:38:46 2001
+++ perl-5.6.1/utils/perldoc.PL	Fri May 11 14:20:57 2001
@@ -315,6 +315,7 @@
     $global_target = (splitdir $s)[-1];   # XXX: why not use File::Basename?
     for ($i=0; $i<@dirs; $i++) {
 	$dir = $dirs[$i];
+	next unless -d $dir; # some dirs in @INC are optional
 	($dir = VMS::Filespec::unixpath($dir)) =~ s!/\z!! if $Is_VMS;
 	if (       ( $ret = check_file $dir,"$s.pod")
 		or ( $ret = check_file $dir,"$s.pm")
